include/controller.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CONTROLLER_H
+#define CONTROLLER_H
+
+#define BTN_A           0x8000
```

Comment:
I think this is the only "enum" / flag defines that are now listed in descending order, so that is kinda weird. are you sure that's the right call? it's not really a big deal so ignore me if you really prefer it this way

Reply:
The idea is that these will eventually be wrapper defines for the original libultra defines, for example:
```
#define BTN_DUP       U_JPAD
#define BTN_DLEFT     L_JPAD
#define BTN_DRIGHT  R_JPAD
#define BTN_DDOWN D_JPAD
```
so the values will be hidden anyway. I dont really have an opinion on reordering in this case either way though

Reply:
well I guess you could think of it being defined left-to-right, which matches English text. But I do think this order is much more natural (A before B, L before R, LEFT before RIGHT)

Reply:
such a western centered view smh